CUSTOM_BUNDLE.md

```diff
@@ -1,13 +1,31 @@
 # Custom bundle
 You can simply make custom bundles yourself, if none of the [distributed packages](https://github.com/plotly/plotly.js/blob/master/dist/README.md) meet your needs, or you want to make a more optimized bundle file with/without specific traces and transforms.
 
-Install plotly.js, move to plotly.js folder then install plotly.js dependencies:
+Clone plotly.js, where the <version> is one of [these](https://github.com/plotly/plotly.js/tags):
+```sh
+git clone --depth 1 --branch <version> https://github.com/plotly/plotly.js.git
+```
+
+Move to plotly.js folder then install plotly.js dependencies:
+```sh
+cd plotly.js
+```
+
+Make sure you have the versions of node/npm that's recommended in the [./CONTRIBUTING.md](./CONTRIBUTING.md#prerequisites) list of Prerequisites.
```

Comment:
How about moving this up to be before `git clone` step?
Also you may link the file to the GitHub. No?

Reply:
No, the whole point is that they clone a version of plotly.js, and then check that versions CONTRIBUTING.md, since the recommended node version has changed.

Reply:
I inlined a compat table, instead of linking to contribution.md. It's much simpler